<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderWindow</class>
 <widget class="QMainWindow" name="OrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OrderWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="data.qrc">:/img/pizza-1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="data.qrc">:/img/pizza-2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="data.qrc">:/img/pizza-3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="data.qrc">:/img/pizza-4.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pizza_1">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>120</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Пицца 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pizza_2">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>120</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Пицца 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pizza_3">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>120</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Пицца 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pizza_4">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>120</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Пицца 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="data.qrc">:/img/pizza-4.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="data.qrc">:/img/pizza-3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="data.qrc">:/img/pizza-1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="data.qrc">:/img/pizza-2.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pizza_set">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>320</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Набор</string>
    </property>
   </widget>
   <widget class="OrderTable" name="order_table">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>270</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="order_send">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>320</y>
      <width>281</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string> Оформить заказ</string>
    </property>
   </widget>
   <widget class="QToolButton" name="pizza_remove">
    <property name="geometry">
     <rect>
      <x>553</x>
      <y>299</y>
      <width>26</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>pizza_1</zorder>
   <zorder>pizza_2</zorder>
   <zorder>pizza_3</zorder>
   <zorder>pizza_4</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_6</zorder>
   <zorder>label_5</zorder>
   <zorder>pizza_set</zorder>
   <zorder>order_table</zorder>
   <zorder>order_send</zorder>
   <zorder>pizza_remove</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OrderTable</class>
   <extends>QTableWidget</extends>
   <header location="global">ordertable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="data.qrc"/>
 </resources>
 <connections/>
</ui>
